src/Universe/PlanetRepository.php

```diff
@@ -55,4 +57,185 @@ public function getPlanetCount(int $userId): int
                 'userId' => $userId,
             ])->execute()->fetchOne();
     }
+
+    public function count(): int
+    {
+        return (int) $this->createQueryBuilder()
+            ->select("COUNT(id)")
+            ->from('planets')
+            ->execute()
+            ->fetchOne();
+    }
+
+    public function countWithUser(): int
+    {
+        return (int) $this->createQueryBuilder()
+            ->select("COUNT(id)")
+            ->from('planets')
+            ->where('planet_user_id > 0')
+            ->execute()
+            ->fetchOne();
+    }
+
+    public function find(int $id): ?array
+    {
+        $data = $this->createQueryBuilder()
+            ->select('*')
+            ->from('planets')
+            ->where('id = :id')
+            ->setParameters([
+                'id' => $id,
+            ])
+            ->execute()
+            ->fetchAssociative();
+
+        return $data !== false ? $data : null;
```

Comment:
can you the Planet class instead of array

Reply:
Done 842e0f7